pkg/storage/system_intake_contract_number_test.go

```diff
@@ -0,0 +1,95 @@
+package storage
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+
+	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/sqlutils"
+	"github.com/cmsgov/easi-app/pkg/testhelpers"
+)
+
+func (s *StoreTestSuite) TestLinkSystemIntakeContractNumbers() {
+	ctx := context.Background()
+
+	const (
+		contract1 = "1"
+		contract2 = "2"
+		contract3 = "3"
+	)
+
+	var createdID uuid.UUID
+
+	// first, create system intake
+	s.Run("create system intake for test", func() {
```

Comment:
Same feedback here around use of `s.Run()` as in the resolver test!

Reply:
Done! Thanks!